query-engine/js-connectors/js/neon-driver-adapter/src/neon.ts

```diff
@@ -110,12 +108,12 @@ export class NeonWSAdapter extends NeonWsQueryable<neon.Pool> implements Connect
     return bindTransaction(new NeonTransaction(connection))
   }
 
-  async close() {
-    this.client.on('error', (e) => console.log(e))
-    if (this.isRunning) {
-      await this.client.end()
-      this.isRunning = false
-    }
+  async connect() {
+    debug('[js::connect]')
+  }
```

Comment:
One would be inclined to add `await this.pool.connect()`, but that wouldn't refresh the pool connections.
Rather, it would select one to use that as a "transactional client".